### ISSUE TITLE
how to use u_net model on grayscale images

### ISSUE THREAD

**Issue by rezadarooei** [yingkaisha/keras-unet-collection#30]
hello dear,
I have a problem when I want to use models on the grayscale images. 
I think it is not possible to use 
`model = models.unet_2d((None, None, 1), [64, 128, 256, 512, 1024], n_labels=2,
                      stack_num_down=2, stack_num_up=1,
                      activation='GELU', output_activation='Softmax', 
                      batch_norm=True, pool='max', unpool='nearest', name='unet')`
as input image size.
how to solve this problem?
Thanks a lot

**Comment by yingkaisha** [comment]
Hi @rezadarooei 

Sorry for coming back to this late.

I have just tested grayscale image cases with your configuration, and it works as expected---there seems no problem.

Would you mind sharing your script/code blocks?

Thank you.

**Comment by rezadarooei** [comment]
Thanks a lot
